src/axom/sidre/docs/sphinx/data_vs_metadata.rst

```diff
@@ -186,36 +186,33 @@ The output of the code is::
 Example 3: Two Views and One Buffer (View Copy)
 -----------------------------------------------
 
-The third example illustrates another  Sidre usage pattern in which multiple 
+The third example illustrates another Sidre usage pattern in which multiple 
 views share the same data in a single buffer. Before we start, we verify that 
 the datastore contains one buffer and that the "A" group contains one view. 
-This is the Sidre buffer, group, and view state that exists 
-at the end of the second example.
-
-We begin by making a copy of the view in group "A" in group "B". The new 
-view is identical in description and data associated with it as the original
-view. In particular, the data in the new view *is the same data* in the
-original view. Recall that Sidre copy operations for groups and views are
-**shallow copy operations**. We verify this, by printing the values of the
-array associated with each view and also the base address of each array.
+This is the Sidre buffer, group, and view state that exists at the end of the 
+previous example.
+
+We begin by making a copy of the group "A" view "aview2" in group "B". The new 
+view is identical in name, description, and data associated with it as the 
+original view. In particular, the data in the new view *is the same data* 
+associated with the original view. Recall that Sidre copy operations for 
+groups and views are **shallow copy operations**. This means that a copy of 
```

Comment:
Thank you for emphasizing that we do shallow copy operations.  It might be worthwhile to put in a brief discussion about how to make a deep copy.

Reply:
I will add another example and document it here.

Reply:
Better: I will add an issue to add a Group::viewDeepCopy() method since I think that will be useful in general.